src/binder/clrprivbindercoreclr.cpp

```diff
@@ -159,8 +159,8 @@ HRESULT CLRPrivBinderCoreCLR::BindUsingPEImage( /* in */ PEImage *pPEImage,
                 {
                     if (pCoreCLRFoundAssembly->GetIsInGAC())
```

Comment:
I think we do. In case the bind is successfull but not to a TPA assembly we do not want to set ppAssembly .